README.md

```diff
@@ -2,45 +2,154 @@
 
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cpp/cpp%20-%20client%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1611&branchName=master)
 
-This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [developer docs](https://azure.github.io/azure-sdk-for-cpp).
 
 ## Getting started
 
-To get started with a library, see the **README.md** file located in the library's project folder. You can find these library folders grouped by service in the `/sdk` directory.
+For the best development experience, we recommend developers use [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies. You can also use any other text editor of your choice, such as [VS Code](https://code.visualstudio.com/), along with the command line for building your application with the SDK.
 
-For tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developers](https://azure.github.io/azure-sdk-for-cpp).
+You can find additional information for specific libraries by navigating to the appropriate folder in the `/sdk` directory. See the **README.md** file located in the library's project folder, for example, the [Azure Storage client library](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage#azure-storage-client-library-for-c).
 
-## Packages available
-Each service might have a number of libraries available from each of the following categories:
-* [Client - New Releases](#client-new-releases)
-* [Client - Previous Versions](#client-previous-versions)
+For API reference docs, tutorials, samples, quick starts, and other documentation, go to [Azure SDK for C++ Developer Docs](https://azure.github.io/azure-sdk-for-cpp).
+
+### Download & Install the SDK
+
+The easiest way to acquire the C++ SDK is leveraging vcpkg package manager. You will need to install [Git](https://git-scm.com/downloads) before getting started.

```

Comment:
@RickWinter I removed the link to the vcpkg repo from here, since I link it later below and also removed example commands like `> .\vcpkg\vcpkg install [packages to install]`, leaving just the concrete ones.

Let me know if this is what you had in mind, and if you have any other suggestions.

Reply:
Seems reasonable.  We should give enough detail to onboard and unblock the customer while not becoming the vcpkg documentation.